### PR TITLE
[bitnami/grafana-operator] Add image pull secrets to grafana template

### LINKED ISSUE #5762: grafana-operator chart does not pass global.imagePullSecrets to grafana deployment
**Which chart**:
grafana-operator 0.3.1

**Describe the bug**
global.imagePullSecrets from values.yaml is not used when pulling bitnami/grafana image even though global.imageRegistry is used.  This prevents the image from being pulled from the private registry specified in global.imageRegistry.

**To Reproduce**
1. add values to values.yaml
```
grafana-operator:
  enabled: true
  global:
    imagePullSecrets: [ regcred ]
    imageRegistry: some.registry.net
```
2.helm install name [chart]

**Expected behavior**
Deployment grafana-deployment should have `spec.containers.imagePullSecrets` and it's value should be `regcred`

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.2", GitCommit:"23dd3af5e19a02d4f4baa5b2f242645a1a3af629", GitTreeState:"dirty", GoVersion:"go1.15.5"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.4", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"clean", BuildDate:"2021-02-21T20:23:45Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"darwin/amd64"}
```

**Additional context**
We're operating in an air-gapped environment so everything has to pull from a private registry.  The grafana operator pulls just fine using the registry and image pull secret but it's this additional image that seems to be the problem.

### PR DESCRIPTION
**Pull request by JamesCacioppo**
**Description of the change**

This change allows `global.imagePullSecrets` to be templated into the grafana deployment so when pulling the image from a private registry using `imageRegistry` the pull does not fail due to authentication.

**Benefits**

This allows `global.imagePullSecrets` to work as described in the readme and prevents deployments from failing when using `global.imageRegistry`

**Possible drawbacks**

**Applicable issues**

  - fixes #5762 

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/grafana-operator/templates/grafana.yaml [hunk 1]
```diff
@@ -38,6 +38,8 @@ spec:
     {{- if .Values.commonAnnotations }}
     annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 6 }}
     {{- end }}
+  serviceAccount:
+    {{- include "common.images.pullSecrets" (dict "images" (list .Values.operator.image .Values.grafana.image .Values.grafanaPluginInit.image) "global" .Values.global) | nindent 4 }}
```
Comment (juan131) [inline bitnami/grafana-operator/templates/grafana.yaml:42]: Why did you add the image pull secrets under the "serviceAccount" section?

**Review by juan131 — COMMENTED** [review]
Thanks for this contribution!

File: bitnami/grafana-operator/templates/grafana.yaml (hunk 1 above)
Comment (JamesCacioppo) [reply bitnami/grafana-operator/templates/grafana.yaml:42]: @marcosbc Suggested I give it a try.  See https://github.com/bitnami/charts/issues/5762 for more details.

Comment (juan131) [reply bitnami/grafana-operator/templates/grafana.yaml:42]: Apparently it's sth supported. See https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-image-pull-secret-to-service-account
I wasn't aware of this feature.

**Review by juan131 — APPROVED** [review]
LGTM
